src/Form.js

```diff
@@ -42,6 +43,19 @@ const Form = () => {
             {errors.password && <p className="error">{errors.password}</p>}
           </div>
         </div>
+        <div>
+          <label>Telephone</label>
+          <div>
+            <input
+              className={`${errors.telephone && "inputError"}`}
```

Comment:
This was more or less the test case to add an arbitrary field and remove any coupling from the useForm custom hook.